File: api/code-samples/python/eth/others/getRawTransactionByHash.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

transactionHash = "0x5bbcde52084defa9d1c7068a811363cc27a25c80d7e495180964673aa5f47687"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_transaction_by_hash(transactionHash)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/transaction/getRawTransactionByHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const transactionHash =
    "0x5bbcde52084defa9d1c7068a811363cc27a25c80d7e495180964673aa5f47687";

  provider.eth
    .getRawTransactionByHash(transactionHash, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
